PedDetection.md

```diff
@@ -59,30 +59,35 @@ The pedestrian detection system shall be capable of detecting the presence of pe
    1. For thermal processors, the pedestrian detection system manufacturer shall indicate how the sensor adjusts to ambient temperature changes at the installation location.
    1. For video processors, the pedestrian detection system must not be susceptible to horizontal and vertical blooming due to brightly lit objects, such as car headlights, that may obscure image detail and blind the image processor.
-   1. Please describe your system’s anticipated pedestrian detection rate and false alarm rate for this intended application.
+   1. Please describe your system’s anticipated pedestrian detection rate and false alarm rate for this intended application. Please describe your company's method of assessing precision and recall of the system and state overall anticipated accuracy within +-5% of actual results.
 1. Minimum Scene Illumination:  If applicable, the pedestrian detection system manufacturer shall describe the minimum illuminance (in foot-candles) required at the PHB locations for the effective use of its pedestrian detection system. 
 1. Resolution: The minimum resolution required is 640×480.
 1. Variable Frame Rate: Frames per second (fps) shall be user-definable from 1-30 fps. 
 1. Human Detection Distances:  The pedestrian detection system manufacturer shall discuss the human detection distances associated with the appropriate products for CCD’s specific application.  A qualified manufacturer’s engineer or technician is invited to perform a field visit to survey the first four PHB locations where the system will be implemented and provide recommendations on the ideal mounting location(s) for its pedestrian detection system, including the number of detectors it recommends for each site.  Only one of the four locations has a raised median with no landscaping.  Proposed mounting locations include traffic signal poles, traffic signal mast arms, and street light poles with traffic signal mast arms. If possible, pre-existing Astro-Brac camera brackets already purchased by CCD shall be the preferred option for mounting.
-1. Data Collection: The pedestrian detection system shall be capable of collecting speed, count, and presence.  This can include, but is not limited to, the number of times the presence of a pedestrian(s) was detected and average crossing speeds or time utilized for crossing.
+1. Data Collection: The pedestrian detection system shall be capable of collecting speed, count, and presence.  This can include, but is not limited to, the number of times the presence of a pedestrian(s) was detected and average crossing speeds or time utilized for crossing. This can include live and post-processed data on instantaneous speed and time to cross. Please provide information about whether your system supports detection of pedestrian queues forming at the curb. 
```

Comment:
@MatthewJMcAllister keep first sentence, remove "Please provide..." final sentence. Future need outside of current scope.